### ISSUE TITLE
The metric I got is different from that reported in the paper

### ISSUE THREAD

**Issue by bigTexas** [francoisgermain/SpeechDenoisingWithDeepFeatureLosses#2]
First thanks for providing the code. I tried to test the algorithm by running the inference (python senet_infer.py) on the validation dataset. The metrics I obtained is avg_sig=3.785319, avg_bak=3.273069, avg_ovl=3.141790 while the number reported in the paper is: avg_sig=3.86, avg_bak=3.33, avg_ovl=3.22. I'm wondering whether I missed something. I didn't change any code and use exactly the same evaluation code mentioned in the paper. FYI, when I tested on the noisy data (no enhancement is applied), the numbers I got is the same as that in paper. Any idea?

**Comment by francoisgermain** [comment]
Hi.
Thanks for your interest, and sorry to hear about your issue. I am pretty busy at the moment and I no longer have access to a GPU machine. How urgently do you need an answer?
In case that helps in the meantime, I believe I have somewhere the audio files that were generated (and scored) for the paper results. Let me know.

**Comment by bigTexas** [comment]
Thanks for your replying @francoisgermain ! It will be very helpful if you can point me to the already generated enhanced audios for validation dataset. When you get time later, I appreciate if you can run the inference code and recompute the metrics for the generated audio. For your information, the number I got is computed on 1080Ti GPU and none of your code is changed.

**Comment by aleXiehta** [comment]
Hi, 
I have also met this problem when running the inference code using pre-trained weights provided in the repo and generated the enhanced utterances.

The evaluation tool I'm currently using is from [here](https://ecs.utdallas.edu/loizou/speech/software.htm), 
and the scores I've got are:
                                                       | PESQ | CSIG | CBAK | COVL | SSNR
Deep Feature Loss (paper)            |     -     |  3.86  |  3.33   |   3.22  | -
Deep Feature Loss (code)             |  2.58   |  3.80  |  2.72   |  3.19   | 0.60

It seems that the main differences are CBAK and SSNR.
I will be very grateful if you can give me any support for this. Thank you!
